### PR TITLE
Update install button for themes and disable it on Fenix

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-blog/issues/186
Fixes https://github.com/mozilla/addons-blog/issues/187

### CODE REVIEW

File: tests/assets/dynamic-addon-cards.test.js [hunk 1]
```diff
@@ -513,7 +550,7 @@ describe(__filename, () => {
 
         // Update the add-on card.
         await _updateAddonCard(card, {
-          userAgent: userAgentsByPlatform.android.firefox70,
+          userAgent: userAgentsByPlatform.mac.firefox69,
```
Comment (willdurand) [inline tests/assets/dynamic-addon-cards.test.js:553]: This was a mistake.. mozAddonManager is only available on FF Desktop.

File: src/assets/js/dynamic-addon-cards.js [hunk 2]
```diff
@@ -25,10 +25,11 @@
     getFirefoxButton.querySelector('.GetFirefoxButton-callout').remove();
 
     const button = getFirefoxButton.querySelector('.GetFirefoxButton-button');
+    const isStaticTheme = addon.type === 'statictheme';
```
Comment (bobsilverberg) [inline src/assets/js/dynamic-addon-cards.js:28]: Sorry, I probably commented on/asked about this before, but shouldn't we be using constants for important strings like this?

File: src/assets/js/dynamic-addon-cards.js [hunk 3]
```diff
@@ -139,6 +139,7 @@
 
           if (!isIncompatible && clientApp === 'android') {
             const isRecommended =
+              addon.type === 'extension' &&
```
Comment (bobsilverberg) [inline src/assets/js/dynamic-addon-cards.js:142]: Nit: This isn't really part of what contributes to an add-on being Recommended. The code certainly works, but it seems slightly inaccurate. Should this test perhaps be added to the test on line 147 instead? Or perhaps change `isRecommended` to `isInstallable` or something similar?

File: tests/assets/dynamic-addon-cards.test.js [hunk 4]
```diff
@@ -444,6 +458,29 @@ describe(__filename, () => {
         });
       });
 
+      it('disables the install button when add-on is not an extension', async () => {
```
Comment (bobsilverberg) [inline tests/assets/dynamic-addon-cards.test.js:461]: Nit: Should the test description include something about "on android"?

**Review by bobsilverberg — APPROVED** [review]
Looks good, nice work @willdurand. Just a question and a couple of nits.

r+wc

File: src/assets/js/dynamic-addon-cards.js (hunk 2 above)
Comment (willdurand) [reply src/assets/js/dynamic-addon-cards.js:28]: we should eventually but for now, I think it's used twice, which is almost less chars than `ADDON_TYPE_STATICTHEME`

File: src/assets/js/dynamic-addon-cards.js (hunk 3 above)
Comment (willdurand) [reply src/assets/js/dynamic-addon-cards.js:142]: oh that's right.

File: tests/assets/dynamic-addon-cards.test.js (hunk 4 above)
Comment (willdurand) [reply tests/assets/dynamic-addon-cards.test.js]: Yes, thanks.

**Pull request merged**